src/prototypes/card-variations/example/example.twig

```diff
@@ -0,0 +1,100 @@
+<div id="card-variations">
+  <div class="wrapper-container light o-container o-container--prose">
+    <div class="card-one o-media o-container__content">
+      <div class="test">
+        <div class="o-media__object">
+          <div class="c-avatar">
+            <img
+              src="static/media/src/objects/media/demo/megan.png"
```

Comment:
I am not 100% sure but I _think_ if the images exist in the `static/` directory you can use that path. 

For example:
```html
<img src="clouds/cloud-white-left.svg">
```
Would look for the image in `/static/clouds/cloud-white-left.svg`

So I _think_ you can add the `megan.png` file into the `/static/media/` directory then access it via `/media/megan.png`. 

I think. 😅

Reply:
That worked!! Thank you!!